docs/pref-adjust.rst

```diff
@@ -16,19 +16,21 @@ satisfy::
 Usage
 -----
 
+``pref-adjust list``
+    List all types of preferences. No changes will be made to any units.
+``pref-adjust show``
+    Show the preferences of a unit.
 ``pref-adjust all|goth_all|clear_all``
-    Changes/clears preferences for all dwarves.
+    Changes/clears preferences for all units.
 ``pref-adjust one|goth|clear``
     Changes/clears preferences for the currently selected dwarf.
-``pref-adjust list``
-    List all types of preferences. No changes will be made to any dwarves.
 
 
 Examples
 --------
 
 ``pref-adjust all``
-    Change preferences for all dwarves to an ideal.
+    Change preferences for all units to an ideal.
```

Comment:
```suggestion
    Change preferences for all citizens to an ideal.
```

Reply:
when we use "units" in these docs, we mean all units on the map, not just citizens.